packages/taquito-local-forging/test/taquito-local-forging.spec.ts

```diff
@@ -28,17 +31,6 @@ describe('Forge and parse operations default protocol', () => {
     });
   });
 
-  describe('Forge and parse operations kathmandu protocol', () => {
-    const localForger = new LocalForger();
-    kathmanduCases.forEach(({ name, operation, expected }) => {
```

Comment:
Should we replace kathmanduCases with limanetCases here?

Reply:
test suite is added back with limacases